src/editor/plugins/Link/deserialize.js

```diff
@@ -22,10 +23,15 @@ const hasValidTarget = (aEl) => {
 export const linkDeserializer = (editor, el) => {
   let parent = el;
 
-  const children = Array.from(parent.childNodes)
+  let children = Array.from(parent.childNodes)
     .map((el) => deserialize(editor, el))
     .flat();
 
+  if (isEmpty(children)) {
```

Comment:
I would have put this logic inside an if at the beginning of deserializeChildren because it is general behaviour (am I wrong?):

https://github.com/eea/volto-slate/blob/a4646aca793051fb14dfa1da9ba420f810747496/src/editor/deserialize.js#L90-L93